HeliPort/NetworkManager.swift

```diff
@@ -291,4 +291,41 @@ final class NetworkManager {
         //TODO wpa3
         return ITL80211_SECURITY_UNKNOWN
     }
+
+    class func getSecurityType(_ info: ioctl_sta_info) -> itl80211_security {
+        if info.supported_rsnprotos & ITL80211_PROTO_RSN.rawValue != 0 {
+            //wpa2
+            if info.rsn_akms & ITL80211_AKM_8021X.rawValue != 0 {
+                if info.supported_rsnprotos & ITL80211_PROTO_WPA.rawValue != 0 {
+                    return ITL80211_SECURITY_WPA_ENTERPRISE_MIXED
+                }
+                return ITL80211_SECURITY_WPA2_ENTERPRISE
+            } else if info.rsn_akms & ITL80211_AKM_PSK.rawValue != 0 {
+                if info.supported_rsnprotos & ITL80211_PROTO_WPA.rawValue != 0 {
+                    return ITL80211_SECURITY_WPA_PERSONAL_MIXED
+                }
+                return ITL80211_SECURITY_WPA2_PERSONAL
+            } else if info.rsn_akms & ITL80211_AKM_SHA256_8021X.rawValue != 0 {
+                return ITL80211_SECURITY_WPA2_ENTERPRISE
+            } else if info.rsn_akms & ITL80211_AKM_SHA256_PSK.rawValue != 0 {
+                return ITL80211_SECURITY_PERSONAL
+            }
+        } else if info.supported_rsnprotos & ITL80211_PROTO_WPA.rawValue != 0 {
+            //wpa
+            if info.rsn_akms & ITL80211_AKM_8021X.rawValue != 0 {
+                return ITL80211_SECURITY_WPA_ENTERPRISE
+            } else if info.rsn_akms & ITL80211_AKM_PSK.rawValue != 0 {
+                return ITL80211_SECURITY_WPA_PERSONAL
+            } else if info.rsn_akms & ITL80211_AKM_SHA256_8021X.rawValue != 0 {
+                return ITL80211_SECURITY_WPA_ENTERPRISE
+            } else if info.rsn_akms & ITL80211_AKM_SHA256_PSK.rawValue != 0 {
+                return ITL80211_SECURITY_ENTERPRISE
+            }
+        } else if info.supported_rsnprotos == 0 {
+            return ITL80211_SECURITY_NONE
+        }
+        //TODO wpa3
+        return ITL80211_SECURITY_UNKNOWN
+    }
+
```

Comment:
Duplicated code?

Reply:
The existing function uses ioctl_network_info, this one uses ioctl_sta_info, which is the different data structure returned.   I kept the same function and we can refactor as needed in the future.  And yes, the logic for determining what type of security is in use is the same across the two functions.

Reply:
Wouldn't it be easier to create a `ioctl_network_info` with just the security info from `ioctl_sta_info`? Or, we could just create another class/struct that contains the security information and add it to `ioctl_sta_info` and `ioctl_network_info`?

Reply:
I'll defer to the owners of itlwm/x if they want to majorly refactor ioctl_network_info and ioctl_sta_info.  The API interface at the moment returns these as two distinct data structs.  I can do the thin ioctl_network_info wrapper with just the security info on the HeliPort side to remove some of the duplicate code as suggested.

Reply:
CC @Goshin

Reply:
I actually already separated the security information into its own struct in itlwm/x and HeliPort and it seems to be working well, however I will create a pull request after this one.

Reply:
Ok, just pushed the changes to remove the duplicate getSecurityType call and use a stand-in ioctl_network_info.  That said, if you have already separated out the security info into its own struct in itlwm/x, feel free to reject/abandon this PR and just go for that longer term cleaner solution.

Reply:
I would suggest merging `ioctl_network_info` and `ioctl_sta_info`, as they share many same fields and their info is derived from `ieee80211_node`. This can be done afterward, and the PR looks good to me.